integration-tests/contract-security-tickets-join.spec.ts

```diff
@@ -0,0 +1,198 @@
+        const opContract = await op.contract();
+        const opSend = await opContract.methods.default(`(Pair ${opContract.address} \"test\" 1)`).send();
+        await opSend.confirmation()
+      } catch (error: any) {
+        expect(error.message).toContain('Unable to encode parameter');
+      }
+      done();
+    });
+
+    mondaynet('Verify creating ticket is not possible Join ticket - two different ticketers', async (done) => {
+      try {
+        const opJoin = await Tezos.contract.originate({
+          code: `   {   parameter (option (ticket string));
+                        storage unit;
+                        code
+                          {
+                            CAR; # drop storage, but leave input on stack
+                            IF_NONE { FAIL} {};
+                            PUSH nat 4;
+                            PUSH string "test";
+                            TICKET;
+                            PAIR;
+                            JOIN_TICKETS;
+                            IF_NONE { FAIL } {};
+                            DROP;
+                            UNIT;
+                            NIL operation;
+                            PAIR;
+                          };
+                        }`,
+          init: 'None',
+        });
+
+        await opJoin.confirmation();
+        expect(opJoin.hash).toBeDefined();
+        expect(opJoin.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY);
+        const opJoinContract = await opJoin.contract();
+        expect(await opJoinContract.storage()).toBeTruthy();
+
+        const opCaller = await Tezos.contract.originate({
+          code: `   {   parameter address;
+                        storage unit;
+                        code
+                          {
+                            CAR; # drop storage, but leave input on stack
+                            CONTRACT (option (ticket string));
+                            IF_NONE { FAIL } {};
+                            PUSH mutez 0;
+                            PUSH nat 1;
+                            PUSH string "test";
+                            TICKET;
+                            SOME;
+                            TRANSFER_TOKENS;
+                            NIL operation;
+                            SWAP;
+                            CONS;
+                            UNIT;
+                            SWAP;
+                            PAIR;
+                          };
+                        }`,
+          init: 'Unit',
+        });
+
+        await opCaller.confirmation();
+        expect(opCaller.hash).toBeDefined();
+        expect(opCaller.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY);
+        const opCallerContract = await opCaller.contract();
+        expect(await opCallerContract.storage()).toBeTruthy();
+        const opSend = await opCallerContract.methods.default(`(Pair ${opCallerContract.address} \"test\" 1)`).send();
```

Comment:
wrong parameter for methods.default

Reply:
the purpose of the test is to verify that tickets cannot be created by cheating with bogus parameters